### PR TITLE
Added management command to fill last modified date

### PR DESCRIPTION
**Pull request by kkrampa**
@czue

### CODE REVIEW

File: corehq/apps/domain/management/commands/fill_last_modified_date.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                include_docs=False
+            )
+        ])
+        return filter(lambda x: 'last_modified' not in x, docs)
+
+    def handle(self, *args, **options):
+        for domain_doc in self._get_domains_without_last_modified_date():
+            print "Updating domain {}".format(domain_doc['name'])
+            domain = Domain.wrap(domain_doc)
+            domain.save()
```
Comment (kkrampa) [inline corehq/apps/domain/management/commands/fill_last_modified_date.py]: @czue There is probably 7000 domains without ```last_modified``` on production. Is it ok to call save for each of them or it should be switched to bulk update?

Comment (czue) [inline corehq/apps/domain/management/commands/fill_last_modified_date.py]: I think this is fine given how few there are

File: corehq/apps/domain/management/commands/fill_last_modified_date.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    def _get_domains_without_last_modified_date(self):
+        docs = iter_docs(Domain.get_db(), [
+            domain['id']
+            for domain in Domain.view(
+                "domain/domains",
+                reduce=False,
+                include_docs=False
+            )
+        ])
+        return filter(lambda x: 'last_modified' not in x, docs)
```
Comment (czue) [inline corehq/apps/domain/management/commands/fill_last_modified_date.py]: do you not also need to check for `last_modified=None`?

**Comment by czue** [comment]
no blockers. assume you'll update if needed and also follow up with interrupt on getting this run

**Pull request merged**